documenters_aggregator/spiders/ccc.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import scrapy
+import re
+
+from datetime import datetime
+from pytz import timezone
+
+
+class CccSpider(scrapy.Spider):
+    name = 'ccc'
+    allowed_domains = ['http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
+    start_urls = ['http://http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
+
+    # for each link
+    #   parse 
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        for link in response.css('a.Eventslink::attr(href)').extract():
+            next_url = "http://www.ccc.edu" + link
+            return scrapy.Request(next_url, callback=self.parse_event_page())
+
+    def parse_event_page(self, response):
+            yield {
+                '_type': 'event',
+                'id': self._parse_id(),
+                'name': self._parse_name(),
```

Comment:
Try `self._parse_name(response)`. The _parse_name function probably needs the response to know what to parse.